index.html

```diff
@@ -1,13 +1,15 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
Along with this you should add a font configuration in your css configuration to have fallback fonts.

```css
p {
  font-family:"Times New Roman", Times, serif;
}
```
(Just an example)